src/models/nodeRuntime.ts

```diff
@@ -1,7 +1,7 @@
-export type NodeRuntime = "nodejs16.x" | "nodejs18.x";
-export const NODEJS_MAJOR_VERSIONS = ["16", "18"];
-export const DEFAULT_NODE_RUNTIME: NodeRuntime = "nodejs16.x";
+export type NodeRuntime = "nodejs16.x" | "nodejs18.x" | "nodejs20.x";
+export const NODEJS_MAJOR_VERSIONS = ["16", "18", "20"];
```

Comment:
Delete this. It's not used anywhere in code.

Reply:
Done